src/fidesops/util/oauth_util.py

```diff
@@ -25,6 +26,9 @@
 )
 from fidesops.schemas.oauth import OAuth2ClientCredentialsBearer
 
+if TYPE_CHECKING:
+    from fidesops.models.fidesops_user import FidesopsUser
```

Comment:
I was getting a circular import error here from pylint. Since `FidesopsUser` is only used as a type, using `if TYPE_CHECKING` allows it to only be imported when mypy is running, and not during runtime therefore avoiding the circular import.

Reply:
that makes sense, thanks for the explanation